database_schemas/DB_generation_schema.sql

```diff
@@ -1,131 +1,176 @@
--- Database setup file for RABIT
+ -- Database setup file for RABIT
 -- ==============================
 -- WARNING!
 -- Running this file will DELETE ALL EXISTING DATA if the database is already configured
 -- ==============================
+SET foreign_key_checks = 0;
 
-SET FOREIGN_KEY_CHECKS = 0;
 DROP TABLE IF EXISTS rabit_user;
 DROP TABLE IF EXISTS upload;
 DROP TABLE IF EXISTS file_pointer;
 DROP TABLE IF EXISTS plot_collection;
 DROP TABLE IF EXISTS base_parameter;
 DROP TABLE IF EXISTS corner_plot;
 DROP TABLE IF EXISTS parameter_config;
-SET FOREIGN_KEY_CHECKS = 1;
+DROP TABLE IF EXISTS dataset_config;
+DROP TABLE IF EXISTS dataset_sigma;
+DROP TABLE IF EXISTS dataset_quantile;
 
--- Generate tables and set their primary keys as constraints
+SET foreign_key_checks = 1;
 
+-- Generate tables and set their primary keys as constraints
 CREATE TABLE rabit_user
-(
-    user_id      CHAR(36)   NOT NULL,
-    display_name VARCHAR(255) NOT NULL,
-    email        VARCHAR(255) NOT NULL
-);
+  (
+     user_id      CHAR(28) NOT NULL,
+     display_name VARCHAR(255) NOT NULL,
+     email        VARCHAR(255) NOT NULL
+  );
 
 ALTER TABLE rabit_user
-    ADD CONSTRAINT rabit_user_pk PRIMARY KEY (user_id);
+  ADD CONSTRAINT rabit_user_pk PRIMARY KEY (user_id);
 
 CREATE TABLE upload
-(
-    upload_id       CHAR(36) NOT NULL,
-    user_id         CHAR(36) NOT NULL,
-    upload_datetime DATETIME
-);
+  (
+     upload_id       CHAR(36) NOT NULL,
+     user_id         CHAR(36) NOT NULL,
```

Comment:
```suggestion
     user_id         CHAR(28) NOT NULL,
```
`user_id` should have type `CHAR(28)`